dispatch/bin/dispatch-admin

```diff
@@ -4,6 +4,10 @@ import sys
 import dispatch
 import coverage
 
+from dispatch.resources import messages as messages
```

Comment:
You can define the text at the top of the file with `""" """` then use it as `__doc__` for simplicity